Sources/CoreMetrics/SystemMetrics.swift

```diff
@@ -0,0 +1,98 @@
+//===----------------------------------------------------------------------===//
+//
+// This source file is part of the Swift Metrics API open source project
+//
+// Copyright (c) 2018-2019 Apple Inc. and the Swift Metrics API project authors
```

Comment:
I think this should be picked up for all files in a separate PR CC @tomerd

Reply:
Technically the rule we are told at  is a more involved: When you touch a file in 2020, you should bump the year to 2020. If a file was not touched in 2020 then it can remain whatever year it was... I don't think many teams follow this strictly in OSS to be honest, but overall that's the latest guidance I've been told 🤔 

Here 2020 would be good in any case I think?